src/05-derivatives/02-common-data-types.md

```diff
@@ -8,7 +8,7 @@ Template:
 <pipeline_name>/
     sub-<participant_label>/
         anat|func|dwi/
-        <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+        <source_keywords>[_space-<space>][_res-<index>][_den-<label>][_desc-<label>]_<suffix>.<ext>
```

Comment:
since the format only specifies `anat|func|dwi`, does this mean that `res` and `den` don't apply to `meg|eeg|ieeg|beh`?

Reply:
Updated to include the additional modalities.

I'm also thinking this should be updated when the BEP about connectivity is merged in, as resolution is something connectivity analyses surely care about.